src/plugins/vis_type_timeseries/public/application/components/color_rules.tsx

```diff
@@ -100,9 +120,14 @@ export class ColorRules extends Component<ColorRulesProps> {
 
   handleOperatorChange = (item: ColorRule) => {
     return (options: Array<EuiComboBoxOptionOption<string>>) => {
+      const selectedOperator: ColorRulesOperator | void = colorRulesOperatorsList.find(
+        (operator) => options[0]?.value === operator.method
+      );
+      const value = (selectedOperator && selectedOperator.value) ?? null;
```

Comment:
nit: for some places you use `options[0]?.value` for some`(selectedOperator && selectedOperator.value)`
I vote for one style =)